source/index.txt

```diff
@@ -128,6 +127,14 @@ Learn what changes you may need to make to your application to upgrade
 driver versions in the :ref:`Upgrade Driver Versions <csharp-upgrade-driver>`
 section.
 
+Entity Framework Provider
+-------------------------
+
+The MongoDB Entity Framework Provider is an object-relational mapper (ORM) that lets you
+use Microsoft's Entity Framework to work with MongoDB data in an object-oriented way.
```

Comment:
suggestion:

```suggestion
The MongoDB Entity Framework Provider is an object-relational mapper (ORM) that lets you
use Microsoft's Entity Framework to work with MongoDB data. ORM provides an object-oriented interface for data management.
```

Reply:
good suggestion. i wasn't crazy about that sentence to begin with